src/app/shared/Header/Header.scss

```diff
@@ -1 +1,400 @@
+                        background-color: #FFFFFF59;
+                        border: none;
+                        width: 6%;
+                        height: 35%;
+
+
+
+                    }
+                }
+            }
+
+        }
+    }
+
+    .for-picturesss {
+        display: flex;
+        flex-direction: row;
+        margin-top: 0.4%;
+        column-gap: 0.4%;
+
+        .for-foto1 {
+            width: 34%;
+            height: 122px;
+            background: url("../../../assets/Images/firstsidepicture.png");
+            background-size: cover;
+            background-position-x: center;
+            border: none;
+
+            .div-for-foto {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                flex-direction: column;
+                row-gap: 10px;
+
+                .p1-for-foto1 {
+                    color: #dbd9d9;
+                    font-family: 'Inter', sans-serif;
+                    font-size: 12px;
+                }
+
+                .p2-for-foto1 {
+                    font-family: 'Inter', sans-serif;
+                    color: #dbd9d9;
+                    font-size: 20px;
+                }
+            }
+        }
+
+
+
+
+        .for-foto2 {
+            background: url("../../../assets/Images/firstsidepicture.png");
+            background-size: cover;
+            background-position-x: center;
+            width: 34%;
+            height: 122px;
+            border: none;
+
+            .div-for-foto {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                flex-direction: column;
+                row-gap: 10px;
+
+                .p1-for-foto2 {
+                    color: #dbd9d9;
+                    font-family: 'Inter', sans-serif;
+                    font-size: 12px;
+                }
+
+                .p2-for-foto2 {
+                    font-family: 'Inter', sans-serif;
+                    color: #dbd9d9;
+                    font-size: 20px;
+                }
+            }
+        }
+
+        .for-foto3 {
+            background: url("../../../assets/Images/firstsidepicture.png");
+            background-size: cover;
+            background-position-x: center;
+            width: 34%;
+            height: 122px;
+            border: none;
+
+            .div-for-foto {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                flex-direction: column;
+                row-gap: 10px;
+
+                .p1-for-foto3 {
+                    color: #dbd9d9;
+                    font-family: 'Inter', sans-serif;
+                    font-size: 12px;
+                }
+
+                .p2-for-foto3 {
+                    font-family: 'Inter', sans-serif;
+                    color: #dbd9d9;
+                    font-size: 20px;
+                }
+            }
```

Comment:
you're breaking the DRY principle
these styles are just like styles from line 307 till 334